### DISCUSSION TITLE
Welcome to opentelemetry-collector Discussions!

### DISCUSSION THREAD

**Discussion by bogdandrutu** [open-telemetry/opentelemetry-collector#2346] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ochauhan** [comment]
I have existing Metrics in JSON format:
{
 "entries": {
          "aborts": {
            "value": 0
          },
          "avgCycles": {
            "value": 0
          },
          "eventType": {
            "description": "ACCESS_POLICY_COMPLETED"
          },
          "failures": {
            "value": 0
          },
          "maxCycles": {
            "value": 0
          },
          "minCycles": {
            "value": 0
          },
          "tmName": {
            "description": "/Common/_sys_APM_ExchangeSupport_OA_BasicAuth"
          },
          "priority": {
            "value": 500
          },
          "totalExecutions": {
            "value": 0
          }
        }
      }
Now I want to convert this to OpenTelemetry Metrics format. How to convert these string values to Otel Metrics or what is the better way to handle the string/text values?

**Reply by bogdandrutu** [reply]
@ochauhan you should write your own custom receiver that does the translation :) Essentially implement a pdata.MetricsMarshaler

**Comment by rohitvyas06** [comment]
Hi Guys I am new to OpenTelemetry, Actually I want to implement Auto-instrumentation with Scala. If I will  use javaagent jar so it will work  or not.

**Reply by bogdandrutu** [reply]
Please ask this question in the Java repository

**Reply by jkwatson** [reply]
https://github.com/open-telemetry/opentelemetry-java-instrumentation/discussions would be the appropriate place to post questions
